### PR TITLE
Adding Hamming Code

### PR DESCRIPTION
**Pull request by memeghaj10**
I am creating a pull request for...

Adding Hamming Code Error Detection while transfer of data bits from sender to user.

Hope you would like and accept my pull request!

Please, do inform if any type of suggestions/changes has to be made. I would ahppily do it.

### CODE REVIEW

File: algorithms/Networks/Hamming_Code/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Hamming Code
```
Comment (abranhe) [inline algorithms/Networks/Hamming_Code/README.md:1]: Remove this file.

To add docs, check this repo: https://github.com/AllAlgorithms/algorithms

File: readme.md [hunk 2]
```diff
@@ -285,9 +285,10 @@ just a paper and pencil.
```
Comment (abranhe) [inline readme.md:285]: Don't change anything in the readme, can you revert this??

**Review by abranhe — CHANGES_REQUESTED** [review]
Thanks so much for your contribution to [The All ▲lgorithms Project](https://github.com/allalgorithms). Without people like you submitting Pull Requests we couldn't run this project. You rock!

Please fix the comments left, in other to merge the pull request.

File: readme.md (hunk 2 above)
Comment (memeghaj10) [reply readme.md]: Yes sure, I have reverted the changes.

File: algorithms/Networks/Hamming_Code/README.md (hunk 1 above)
Comment (memeghaj10) [reply algorithms/Networks/Hamming_Code/README.md]: Ok, no problem. I have removed the README.md file for this section.

**Review by abranhe — APPROVED** [review]
Thanks so much for your contribution to [The All ▲lgorithms Project](https://github.com/allalgorithms). Without people like you submitting Pull Requests we couldn't run this project. You rock!

**Pull request merged**